hummingbot/connector/gateway_EVM_AMM.py

```diff
@@ -613,9 +614,78 @@ async def _update_balances(self, on_interval=False):
             self._in_flight_orders_snapshot = {k: copy.copy(v) for k, v in self._in_flight_orders.items()}
             self._in_flight_orders_snapshot_timestamp = self.current_timestamp
 
+    async def execute_cancel(self, trading_pair: str, client_order_id: str) -> str:
+        await self._general_cancel(client_order_id, None)
+
+    async def _general_cancel(self, order_id: str, cancel_age: Optional[int] = None) -> str:
+        try:
+            tracked_order = self._in_flight_orders.get(order_id)
+            if tracked_order is None:
+                self.logger().error(f"The order {order_id} is not tracked. ")
+                raise ValueError
+
+            # if cancel_age is included, then cancel only if the age of the order
+            # is geq to the cancel_age
+            if cancel_age is not None:
+
+                if (self.current_timestamp - tracked_order.creation_timestamp).total_seconds() < cancel_age:
+                    return
```

Comment:
Idiomatic Python prefers explicit rather than implicit statements (see: PEP20). If this function sometimes returns the order ID, and sometimes don't - you should write `return None` to explicitly say you're returning None to indicate the order is not being canceled.